README.Rmd

```diff
@@ -52,100 +52,132 @@ relational data models.
-These users can benefit from a better data organization. 
-The separation into multiple tables helps data quality but poses a different challenge: for each flight, the location of the origin airport, or the details on the airplane, are not available immediately but must be *joined*/merged:
+In {dm} the basic element is [a `dm` object](https://krlmlr.github.io/dm/articles/dm-class-and-basic-operations.html).
+You can create it with `cdm_nycflights13()` for the example data.
+After that you can use the links between the tables as often as you wish - without explicitly referring to the relations ever again.
+The task of joining four tables (`flights`, `airlines`, `planes` and `airports`) boils down to:
 
-```{r join}
+```{r}
+cdm_nycflights13() %>%
+  cdm_flatten_to_tbl(start = flights)
+```
+
+In contrast, using the classical {dplyr} notation you need three `left_join()` calls to merge the `flights` table gradually to `airlines`, `planes` and `airports` tables to create one wide dataframe.
+
+```{r}
 library(tidyverse)
 library(nycflights13)
 
-flights %>% 
-  select(month, day, origin, tailnum) %>% 
-  left_join(airports, by = c("origin" = "faa")) %>% 
-  left_join(planes, by = "tailnum")
+flights %>%
+  left_join(airlines, by = "carrier") %>%
+  left_join(planes, by = "tailnum") %>%
+  left_join(airports, by = c("origin" = "faa"))
 ```
 
-This can result in long and inflated pipe chains full of `left_join()`, `anti_join()` and other forms of merging data. 
+You can find more information and important terms to jump-start working with {dm} in the article ["Introduction to Relational Data Models""](dm-introduction-relational-data-models.html).
+
+### The Advantages in Brief
 
-{dm} offers a more elegant and shorter way to combine values by establishing key relations (see next section [Good to Know](#good-to-know)) while augmenting {dplyr}/{dbplyr} workflows. 
+The separation into multiple tables achieves several goals:
 
-You can have the best of both worlds: Manage your data with {dm} as linked tables, then flatten multiple tables into one for your analysis with {dplyr} on an as-needed basis.
+- **Avoid repetition, conserve memory**: the information related to each airline, airport, and airplane are stored only once
+    - name of each airline
+    - name, location and altitude of each airport
+    - manufacturer and number of seats for each airplane
+- **Improve consistency**: for updating any information (e.g. the name of an airport), it is sufficient to update in only one place
+- **Segmentation**: information is organized by topic, individual tables are smaller and easier to handle
 
-## Good to Know 
+## Good to Know
 
 Multiple, linked tables are a common concept in database management.
 Since many R users have a background in other disciplines, we present five important terms in relational data modeling to jump-start working with {dm}.
 
-### 1) Model
+### 1) Tables
```

Comment:
Perhaps:

```suggestion
### 1) Data frames and tables
```

It's tricky that "table" has slightly different semantics in other fields.